.babelrc

```diff
@@ -2,36 +2,45 @@
   "presets": [
     "@babel/typescript",
     "@babel/react",
-    ["@babel/env", {
-      "useBuiltIns": "usage"
-    }]
+    [
+      "@babel/env",
+      {
+        "useBuiltIns": "usage"
+      }
+    ]
   ],
   "plugins": [
     ["@babel/plugin-proposal-decorators", { "legacy": true }],
     "@babel/plugin-proposal-class-properties",
-    ["relay", { "artifactDirectory": "./src/__generated__" }],
+    "relay",
     [
       "styled-components",
       {
         "ssr": true
       }
     ],
-    ["transform-rename-import", {
-      "original": "..\/src\/__generated__",
-      "replacement": "$1.\/__generated__"
-    }],
-    ["module-resolver", {
-      "extensions": [".js", ".jsx", ".ts", ".tsx"],
-      "root": ["./src"],
-      "alias": {
-        "storybook": "./src/__stories__",
-        "Analytics": "./src/Analytics",
-        "Assets": "./src/Assets",
-        "Components": "./src/Components",
-        "Styleguide": "./src/Styleguide",
-        "package.json": "./package.json"
+    [
+      "transform-rename-import",
+      {
+        "original": "../src/__generated__",
+        "replacement": "$1./__generated__"
+      }
+    ],
```

Comment:
I wonder if this is still needed?

Reply:
Probably, as this is more of a custom build setup decision we made, outside of relay's workings, right?

Reply:
I remember a pathing hiccup prompted this, but am unsure exactly what that was. I'll try removing it in a separate branch and see what happens.